_posts/2019-06-13-omero-5-5-0.md

```diff
@@ -0,0 +1,82 @@
+separate repositories, each with a new [Gradle](https://gradle.org) build system:
+
+- https://github.com/ome/omero-dsl-plugin
+- https://github.com/ome/omero-model
+- https://github.com/ome/omero-common
+- https://github.com/ome/omero-romio
+- https://github.com/ome/omero-renderer
+- https://github.com/ome/omero-server
+- https://github.com/ome/omero-blitz
+- https://github.com/ome/omero-gateway-java
+- https://github.com/ome/omero-blitz-plugin
+- https://github.com/ome/omero-insight
+- https://github.com/ome/omero-matlab
+- https://github.com/ome/omero-javapackager-plugin
+- https://github.com/ome/omero-api-plugin
+
+This has the goal of enabling more fine-grained releases.
+A new restriction is that the names of server configuration properties
+may include only letters, numbers and the symbols "`.`", "`_`", "`-`".
+
+New plugins like omero-artifact-plugin allow reducing the boilerplate
+code in the build scripts of the decoupled repositories.
+Though initially disruptive, we hope this modernization and modularization will ease participation in the development of OMERO.
+
+Beyond the changes for building OMERO, the distribution of [OMERO.server](https://hub.docker.com/r/openmicroscopy/omero-server) and [OMERO.web](https://hub.docker.com/r/openmicroscopy/omero-web-standalone/) as Docker images is now considered production quality. Examples for using these images in various configurations are available under [omero-deployment-examples](https://github.com/ome/omero-deployment-examples). Both images will be updated with every OMERO release, and will also be updated with releases of the embedded components and plugins as necessary.
+
+## User/client improvements
+
+Additionally, this release improves the Web interface when OMERO is opened to the public
+and contains some useful CLI improvements.
+
+- web:
+  - Introduce Advanced Search to allow and/or search options
+  - Fix description in new Project, Dataset etc.
+  - Allow user to create new Map Annotations for multiple selected objects at once
+  - Fix date display
+  - Remove / from 3rdparty in ome.viewportImage.js
+  - Remove usage of deprecated calls
+  - Remove apache config
+  - Do not break display of Tag dialog when large font size is configured in browser
+  - Disable refresh button while existing refresh in progress
+  - Improve public user support
+  - Add ability to customize incorrect login text
+
+- cli:
+  - Disable foreground indexing
+  - Improve logging of error when importing data via cli command
+  - Clearly indicate empty log files when running a diagnostic
+  - Fix bug when running `config load` passing a directory instead of a file
+  - Add option to delete keys from map annotations
+  - Add error code discovery
+  - Deprecate the CLI upload module and plugin
+
+
+The software is available at [archived downloads](https://downloads.openmicroscopy.org/omero/5.5.0). Clients are no longer available from that location.
```

Comment:
```suggestion
The software is available at [archived downloads](https://downloads.openmicroscopy.org/omero/5.5.0). Clients are no longer available from that location since they will be released more frequently than the server.
```